pkgs/development/python-modules/coqui-trainer/default.nix

```diff
@@ -27,20 +26,21 @@ buildPythonPackage {
     hash = "sha256-NsgCh+N2qWmRkTOjXqisVCP5aInH2zcNz6lsnIfVLiY=";
   };
 
+  patches = [ ./conqui-trainer.patch ];
+
   propagatedBuildInputs = [
     coqpit
     fsspec
-    pytorch
+    pytorch-bin
```

Comment:
I did this dance twice now, it is annyoing, because whenever a dependency switches between bin and src I have to follow. :facepalm: